lib/commands/context.js

```diff
@@ -396,7 +396,8 @@ async function openNewPage () {
 }
 
 extensions.typeAndNavToUrl = async function () {
-  this.setCurrentUrl(this.caps.safariInitialUrl || `http://127.0.0.1:${this.opts.port}/welcome`);
+  let address = (this.opts.address === '0.0.0.0') ? '127.0.0.1' : this.opts.address;
```

Comment:
i ran into an issue with trying to actually access `0.0.0.0` on windows, so i think it fails there. but this is for ios and i think it is successful. probably works if we just do `let address = this.opts.address`?